app/src/main/res/layout/fragment_faq.xml

```diff
@@ -5,7 +5,17 @@
     android:layout_height="match_parent"
     android:background="@color/white"
     android:orientation="vertical">
-
+    <TextView
+        android:id="@+id/faq_expand_label"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:text="@string/expand_all_label"
+        android:clickable="true"
+        android:focusable="true"
+        android:gravity="end"
+        android:textColor="@color/colorPrimaryDark"
```

Comment:
Can we make it black. I think it will provide good contrast. What do you think?

Reply:
@harsh-2711  making it black would make it look like plain text. Red kind of gives feel of clicking it. Just like questions headings

Reply:
OK then